regression/cbmc-shadow-memory/var-assign1/main.c

```diff
@@ -1,10 +1,11 @@
 #include <assert.h>
+#include <stdbool.h>
```

Comment:
not needed anymore

Reply:
I guess we'll have a test with `_Bool` elsewhere?

Reply:
Removed.

Yes, there is a test for `_Bool` types, but it will be added later when shadow memory with `_Bool` typed values will be crystallised.